missingness/missingness.Rmd

```diff
@@ -0,0 +1,235 @@
+        if( question == 28 ){
+          # choosing Yes triggers 28b (file upload), 28c, 28d, 28e
+          if( data[i,idx] == levels( data[,idx] )[1] ){
+            update_special(i, "28c")
+            update_special(i, "28d")
+            update_special(i, "28e")
+          }
+        } 
+        
+        if( question == 34 ){
+          # Yes, only Aedes aegypti triggers 34b
+          if( data[i,idx] == levels( data[,idx] )[1] ){
+            update_special(i, "34b")
+          } 
+          # Yes, only Aedes albopictus triggers 34c
+          if( data[i,idx] == levels( data[,idx] )[2] ){
+            update_special(i, "34c")
+          }
+          # Yes, both triggers both 34b and 34c
+          if( data[i,idx] == levels( data[,idx] )[3] ){
+            update_special(i, "34b")
+            update_special(i, "34c")
+          }
+        } 
+        
+        if( question == 35 ){
+          # choosing None triggers 35b
+          if( !is.na(data[i, idx[6]]) & data[i, idx[6]] == "Yes"){
+            update_special(i, "35b")
+          }
+        } 
+        
+        if( question == 37 ){
+          # choosing "Yes, have a threshold that does not require concurrent human cases"
+          # triggers 37b ; no other choice does
+          if( data[i,idx] == levels( data[,idx] )[1] ){
+            update_special(i, "37b")
+          }
+        } 
+        
+        if( question == 41 ){
+          # choosing "Yes" triggers 41b, 41c, 41d (file upload)
+          if( data[i,idx] == levels( data[,idx] )[1] ){
+            update_special(i, "41b")
+            update_special(i, "41c")
+          }
+        } 
+        
+        if( question == 42 ){
+          # choosing "Yes" triggers 42b, 42c (file upload)
+          if( data[i,idx] == levels( data[,idx] )[1] ){
+            update_special(i, "42b")
+          }
+        } 
+        
+        if( question == 43 ){
+          # choosing "Yes" triggers 43b, 43c, 43d, 43e
+          if( data[i,idx] == levels( data[,idx] )[1] ){
+            update_special(i, "43b")
+            update_special(i, "43c")
+            update_special(i, "43d")
+            update_special(i, "43e")
+          }
+        } 
+        
+        if( question == "55b" ){
+          # answering "Yes" to 55b triggers 55c and 55d
+          if( data[i,idx] == levels( data[,idx] )[1] ){
+            update_special(i, "55c")
+            update_special(i, "55d")
+          }
+        }
+      }
+    }
+  }
+}
+
+count_by_country <- summary %>% 
+  group_by(Country,Status) %>% 
+  tally( name="Number skipped") %>% 
+  filter(Status=="skipped") %>% 
+  select(-Status)
+
+list_by_country <- summary %>% 
+  group_by(Country,Question) %>% 
+  filter(Status=="skipped") %>%
+  select(Country,Question) %>% 
+  pivot_wider(names_from=Country, 
+              values_from=Question, 
+              values_fn=list) 
+
+countries <- names( list_by_country)
+skipped <- character( length = length(countries))
+
+for(i in 1:length(countries)){
```

Comment:
I get an error here:
```
Error in `[[.default`(list_by_country[[country]], 1) : 
  subscript out of bounds
```

Reply:
works for me... are you still getting this?

Reply:
should be fixed now by explicitly casting `summary$Status` as a character vector